src/plugins/intel_cpu/tests/unit/jit_kernel_test.cpp

```diff
@@ -269,9 +269,14 @@ struct jit_variable_load_store_test_kernel {
     };
 
     template<size_t N>
-    void test() {
-        kernel_impl<N> kernel;
-        kernel.init();
+    void test(bool is_src = false) {
```

Comment:
Can we pass `jit_kernel::var` type as template parameter instead of explicit flag? This will allow to avoid code duplication

Reply:
Added. Thanks!